hw2/03-count.html

```diff
@@ -9,16 +9,16 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
+  <body class="bg-primary container mt-5">
+    <main class=" bg-light rounded p-4 d-flex flex-column max-sm-75">
+      <h1 class="d-flex justify-content-center">Exercise 03 - Count</h1>
       <hr />
-      <div>
+      <div class="d-flex justify-content-center">
```

Comment:
Content here shouldn't be centered, as it is aligned on the left in the example photo.